ev3dev/sound.py

```diff
@@ -169,9 +204,43 @@ def play_tone(self, frequency, duration_ms, delay_ms=100, volume=100, play_type=
                 play = self.tone([(frequency, duration_ms, delay_ms)])
                 play.wait()
 
-    def play(self, wav_file, play_type=PLAY_WAIT_FOR_COMPLETE):
+    def play_note(self, note, duration, volume=100, play_type=PLAY_WAIT_FOR_COMPLETE):
+        """ Plays a note, given by its name as defined in ``_NOTE_FREQUENCIES``.
```

Comment:
For the musically-inept (e.g. me) could you include explanations to differentiate tone from note? I think tone is arbitrary frequency and notes are named frequencies, but I'm not the expert here.

Reply:
You're 100% right. play_note() is only a convenience call for tone() with the corresponding frequency. The tone() method was already there when I started to collaborate to this module, and it can be useful for arbitrary frequencies. However, I'd suggest to remove the example show how to play a song with tone() since IMHO it can lead to some confusion (you question is a perfect example of such a side effect). I didn't take the decision on my own since not the author of this part of the code.